### ISSUE TITLE
Decorators are failing to be imported when used with Karma and System JS

### ISSUE THREAD

**Issue by alan-agius4** [steelsojka/lodash-decorators#32]
Decorators are failing to be imported when used with Karma and System JS

```

Chrome 59.0.3071 (Windows 10 0.0.0)
Error: Fetch error: 404 Not Found
  Instantiating http://localhost:9876/base/node_modules/lodash-decorators/applicators.js
  Loading http://localhost:9876/base/node_modules/lodash-decorators/after.js
  Loading http://localhost:9876/base/node_modules/lodash-decorators/index.js
  Loading 
```

Issue https://github.com/systemjs/systemjs/issues/1308

**Comment by steelsojka** [comment]
I'm going to create correct barrel files for this instead of adding index to every import.

**Comment by steelsojka** [comment]
Fixed in this commit fda1c7367d27610827fbbd512aa8032299bcd914. Will be in version 4.4.1.

**Comment by alan-agius4** [comment]
Thanks sounds good
